lib/marked_section_deleter.rb

```diff
@@ -0,0 +1,60 @@
+require "services"
+
+class MarkedSectionDeleter
+  def initialize(logger = STDOUT)
```

Comment:
Hmm, good point. How about I rename it to `io`? Unless you have any other suggestions?